spec/unit/type/rpmkey_spec.rb

```diff
@@ -71,5 +71,16 @@
       end
     end
 
+    describe "autorequire" do
+      it "should autorequire a local file" do
+        catalog = Puppet::Resource::Catalog.new
+        file = Puppet::Type.type(:file).new(:name => '/tmp/foo', :content => 'bar' )
+        catalog.add_resource file
+        key = described_class.new(:name => 'DB42A60E', :source => '/tmp/foo', :ensure => :present)
+        catalog.add_resource key
+        key.autorequire.size.should == 1
```

Comment:
the should syntax is deprecated since rspec3, better to use the `expect` syntax:
````ruby
expect(key.autorequire.size).to eq(1)
````